File: matlab/examples/test_gallery/test_gallery_hyphyp1_reentrant.m
% Test gallery_hyphyp1_reentrant with SolverMVA
model = gallery_hyphyp1_reentrant();
solver = SolverMVA(model);
avgTable = solver.getAvgTable();
fprintf('Model: %s\n', model.getName());
disp(avgTable);

File: python/examples/test_gallery/test_gallery_hyphyp1_reentrant.py
#!/usr/bin/env python3
"""Test gallery_hyphyp1_reentrant with SolverMVA"""

import sys
sys.path.insert(0, '../gallery')
from gallery_hyphyp1_reentrant import gallery_hyphyp1_reentrant
from line_solver import SolverMVA

if __name__ == '__main__':
    model = gallery_hyphyp1_reentrant()
    solver = SolverMVA(model)
    avg_table = solver.getAvgTable()
    print(f'Model: {model.getName()}')
    print(avg_table)
